API property template - update for short title (#40672)
mdn/content: files/en-us/mdn/writing_guidelines/page_structures/page_types/api_property_subpage_template/index.md

{{SecureContext_Header}}{{AvailableInWorkers}}{{SeeCompatTable}}{{Deprecated_Header}}{{Non-standard_Header}}

> The **`NameOfTheProperty`** [read-only] property of the \{{domxref("NameOfTheParentInterface")}} interface _<provide concise summary of behavior>_.

Author: ```suggestion
The **`NameOfTheProperty`** [read-only] property of the \{{domxref("NameOfTheParentInterface")}} interface _\<provide concise summary of behavior\>_.
```

> _Start by naming the property (indicating whether or not is is read-only) and what interface it is part of, and then say what it does._

Reviewer: ```suggestion
_Start by naming the property (indicating whether or not it is read-only) and the interface it is part of, and then say what it does._
```

_This should ideally be one or two short sentences._

> _If you need more than a few paragraphs this should be added in a description section before the example section._

Reviewer: ```suggestion
_If you need more than a couple of paragraphs, this should be added in a "Description" section placed before the "Examples" section._
```

## Value

A \{{domxref("SomeDataType" }}.

_Normally just the data type, and allowed values for that data type if relevant._

> _A property that has different setter and getter behaviour would normally cover those in separate sentences._

Reviewer: ```suggestion
_If the property has different setter and getter behaviour, these should normally be covered in separate sentences._
```

_In some cases you may want to say more about what the data type represents._

> _This is acceptable, but should not duplicate information in the description (it is more normal to include information about what the value means in the description)._

Reviewer: ```suggestion
_This is acceptable, but should not duplicate information from the "Description" section (you should include information about what the value means there)._
```

_Note that some property pages are written in the form "Returns a [name of the property type] representing..." but this is not the recommended form.

> Also, some WebIDL extended attributes with specific meanings can be associated with the type. There are standard ways to document them; consult [Information contained in a WebIDL file](/en-US/docs/MDN/Writing_guidelines/Howto/Write_an_api_reference/Information_contained_in_a_WebIDL_file#type_of_the_property) for more information._

Reviewer: Give a couple of examples? I'm assuming you are referring to things like set-like objects that have standard methods, although I couldn't think of any specific property examples, except perhaps "nullable" behavior. From reading the linked resource, it looks like some of these standard ways may require additions to other sections of the page.

Author: I have no idea what this means - it is preexisting content just moved to this spot. I vote we leave it though.

Author: @chrisdavidmills I've accepted all your suggestions except this one - this was a fly-by fix. I am sure this is useful info, but I don't want to expand on it here.

{{SecureContext_Header}}{{AvailableInWorkers}}{{SeeCompatTable}}{{Deprecated_Header}}{{Non-standard_Header}}

> The [read-only] **`NameOfTheProperty`** property of the \{{domxref("NameOfTheParentInterface")}} interface [provide concise summary of behavior].

Author: ```suggestion
The **`NameOfTheProperty`** [read-only] property of the \{{domxref("NameOfTheParentInterface")}} interface _<provide concise summary of behavior>_.
```
